model/host/host.go

```diff
@@ -407,6 +407,14 @@ func (h *Host) IdleTime() time.Duration {
 	return time.Since(h.CreationTime)
 }
 
+func (h *Host) GetAMI() string {
+	if len(h.Distro.ProviderSettingsList) == 0 {
+		return ""
+	}
+	ami, _ := h.Distro.ProviderSettingsList[0].Lookup("ami").StringValueOK()
```

Comment:
I don't really get how the `ProviderSettingsList` works conceptually. How do we know that it must be the first in the list? Is it because hosts are always started in us-east-1?

Reply:
For distros, providerSettingsList can have multiple items because there are definitions for multiple regions, however within the host document we only save the region that is relevant to that host, so it's guaranteed to have only one item.